legacy/plugin.ts

```diff
@@ -89,14 +89,20 @@ export interface PluginMetadata {
 
   // Per-plugin custom metadata
   meta?: {
-    allSubProjectNames?: string[];
+    allSubProjectNames?: string[],
+    versionBuildInfo?: VersionBuildInfo,
   };
 
   // Docker-related fields
   dockerImageId?: any;
   imageLayers?: any;
 }
 
+export interface VersionBuildInfo {
+  gradleVersion: string;
+  metaBuildVersion: { [index: string]: string };
```

Comment:
We are looking to move to return multiple test results in one go from a plugin so I think it may be best to move this meta into each response or maybe re-think what goes in here. As it may be different versions of gradle wrapper per project https://github.com/snyk/snyk/pull/773

@miiila  @orsagie  would love your thoughts here?